### ISSUE TITLE
maxEvents not working in multimonth view

### ISSUE THREAD

**Issue by Pappos4** [fullcalendar/fullcalendar#7270]
### Reduced Test Case

no test case

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

React

### Bug Description

As the title describes I found out that both `dayMaxEventRows` and `dayMaxEvents` are ignored in the multi month view using `multiMonthMaxColumns={1}`.

**Comment by Pappos4** [comment]
Im using `react 18.2.0` and `fullcalendar 6.1.4` (all plugins etc)

**Comment by acerix** [comment]
I believe those settings are not implemented for that view but it could be a useful feature.  Test case showing it doesn't work:

https://codepen.io/acerix/pen/ExdYomY?editors=0010

**Comment by fliespl** [comment]
I would also love to see this happen :)

**Comment by calechko** [comment]
We need this feature as well. Could we use eventLimit or dayMaxEvents similarly to other views?

**Comment by GizaLook** [comment]
I've been playing around trying to do this myself, with little success up to now.
This feature would be very useful on the MultiMonthView.

**Comment by calechko** [comment]
I found that if you adjust the height settings, you can see more events, but it's not quite the same as in the other views. Hope that helps someone until a new version comes out with this feature.

**Comment by urinaldeuce** [comment]
Agreed this feature would be very useful on the multiMonthYear as well.

**Comment by DywiTom** [comment]
Same problem in Angular v17

**Comment by yoancutillas** [comment]
Hi, I am interested in this feature as well :)

**Comment by DywiTom** [comment]
Same issue #7462 
Fixed in [v6.1.11](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.1.11)

**Comment by Pappos4** [comment]
Are there any updates? Is this planned or should someone else work on it? @acerix

**Comment by 555** [comment]
I would also like to see this feature implemented.

**Comment by kira-ru** [comment]
> Yes this would be a very nice feature, in my case i would like to display all events without the "+x More" in the same way as "dayGridMonth" does

dayMaxEventRows and dayMaxEvents still ignored in the multi month view. is there any plans to do something about it? @acerix
